src/main/java/Game.java

```diff
@@ -74,31 +194,107 @@ private static void handleMove(String direction, Player player) {
+			return "You can only proceed by neutralizing the obstacle or going back the direction you came from: " +
+					"<" + oppositeDirection(player.getLastValidDirection()) + ">";
+		}
+		return checkValidityMove(newPos, direction, player);
+	}
+
+	private static String checkValidityMove(Coordinate position, String direction, Player player){
+		if (map.isValidMove(position)) {
+			player.movePlayer(position);
+			player.setLastValidDirection(direction);
+			return map.getDescription(position);
 		} else {
-			player.output("There is nothing " + direction + " of where you are now.");
+			if (map.isValidMove(position)) { return map.getArea(position).getObstacle().getDescription();}
+			return "There is nothing " + direction + " of where you are now.";
 		}
+	}
 
+	private static String handleContainer(String item, Player player, String action) {
+		Area area = map.getArea(player.position()); //.getItems()
+		Container container = (Container) hasItem(area.getItems(), item);
+		StringBuilder toReturn = new StringBuilder();
+		if(container != null && !container.isOpen()) {
+			container.toggle();
+			if(!container.isValidAction(action)) { return "You cannot " + action + " a " + item; }
+			toReturn.append(container.getDescription() + "\n");
+			for(Item containerItem : container.getEntities()) {
+				area.addItem(containerItem);
+			}
+			toReturn.append(area.getItemsDescription());
+		} else {
+			if(container != null && container.isOpen()) {toReturn.append("The " + item + " has already been moved.s"); }
+			else { toReturn.append("I don't see a" + item + " in this room."); }
+		}
+		return toReturn.toString();
 	}
 
 	private static boolean isDirection(String d) {
-		return d.equals("north") || d.equals("south") || d.equals("west") || d.equals("east");
+		return d.equals(north) || d.equals(south) || d.equals(west) || d.equals(east);
 	}
 
-	private static void giveHelpInstructions(Player player) {
-		player.output("I see you are a noob, " + player.getName() + ". You are in the VuORK realm. To interact " +
+	private static String getHelpInstructions(Player player) {
+		return "I see you are a noob, " + player.getName() + ". You are in the VuORK realm. To interact " +
 				"with this realm, you must type a command with the form" +
 				"<action> <item> <proposition> <item>. An example would be 'Attack broom with sword'. For movement," +
 				"there are 2 ways to move: 'move' followed by either 'north', 'south, 'east' or 'west'. You can also " +
-				"simply type 'south' or any other direction as a shortcut. It is important to note that all inputs are " +
-				"NOT case sensitive! For a full list of commands, type 'List'. Good luck and have fun in VuORK");
+				"simply type 'south' or any other direction as a shortcut. By typing 'text'<message> you can send a " +
+				"message to your fellow players. It is important to note that all inputs are NOT case sensitive!" +
+				"For a full list of commands, type 'List' or 'l'. Good luck and have fun in VuORK";
```

Comment:
Very user friendly instructions! I like the inclusion of such detail within your system.